sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -553,6 +535,8 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
         ReloadWorkflowTasks.webpackTask(fastOptJS)
       }.value,
 
+      installWebpackDevServer := npmUpdate.value,
```

Comment:
Next release will be a breaking release since it upgrades the webpack default version.